pybamm/parameters/parameter_values.py

```diff
@@ -617,8 +617,14 @@ def _process_symbol(self, symbol):
                     else:
                         input_data = data
 
+                    # For parameters provided as data we use a cubic interpolant
+                    # Note: the cubic interpolant can be differentiated
                     function = pybamm.Interpolant(
-                        input_data[0], input_data[-1], new_children, name=name
+                        input_data[0],
+                        input_data[-1],
+                        new_children,
+                        interpolator="cubic",
```

Comment:
so the default when creating an interpolant is "linear", but the default when using data is "cubic"?

Reply:
Yeah, I didn't like doing this. I think it would be better to use "linear" all the time, but you can't take the derivative. We could switch to using [InterpolatedUnivariateSpline](https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.InterpolatedUnivariateSpline.html) for the 1D interpolation and just change the degree, then you can, but it's not smooth anyway.